src/WebARKitController.js

```diff
@@ -165,10 +165,10 @@ export default class WebARKitController {
         q += 4;
       }
     }
-    if (this.dataHeap) {
+    /*if (this.dataHeap) {
```

Comment:
lefting out this code because i get:
`Cannot perform %TypedArray%.prototype.set on a detached ArrayBuffer`
Anyway i have to check if i can remove or not.